gcp/api/server.py

```diff
@@ -1049,9 +1051,11 @@ def query_by_version(context: QueryContext,
 
   ecosystem_info = ecosystems.get(ecosystem)
   is_semver = ecosystem_info and ecosystem_info.is_semver
+  supports_ordering = ecosystem_info and ecosystem_info.supports_ordering
 
   bugs = []
   next_page_token = None
+  project = getattr(_ndb_client, 'project', 'oss-vdb')
```

Comment:
nit: Can you make a `get_gcp_project()` function for this line, with the comment from the `LogTraceFilter` that explains this.